server/src/api/bill_exec.py

```diff
@@ -53,12 +53,35 @@ def create_bill(self):
         
         try:
             username = cherrypy.session["username"]
+            #Bill has a title
+            if ((len(title) < 1)):
+                raise ValueError("no title","title")
+            #Bill has a payer
+            if ((len(payer) < 1)):
+                raise ValueError("no payer","payer")
+            #Bill has a total greater than 0
+            if (float(total)<=0):
```

Comment:
Side note: the API documentation specifies total as a `double`, but the app currently returns a `string`.

Reply:
Currently when the total amount is missing, the app sends a total value of empty string `""` and causes `float()` to produce a `ValueError: could not convert string to float`, and the server returns an empty error message to the app due to another bug.